### PR TITLE
Enable --latest flag for the jenkins-plugin-cli in the init container

### PR DESCRIPTION
**Pull request by v1v**
# What this PR does / why we need it

Configure the plugin-installation-manager-tool to skip latest when the whole list of plugins have been defined with the installPlugins.

This should help to bump the version of all the plugins when upgrading the helm.

For instance, if bumping a version of an explicit plugin, then `helm upgrade`, if any of the other explicit plugins have been released with a new version, the init container will fail if all plugins were not bumped. 

It adds the latest flag -> https://github.com/jenkinsci/plugin-installation-manager-tool#cli-options

# Which issue this PR fixes

- https://github.com/jenkinsci/plugin-installation-manager-tool/issues/318

# Special notes for your reviewer

# Checklist

- [x] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] CHANGELOG.md was updated

### CODE REVIEW

File: charts/jenkins/tests/jenkins-controller-statefulset-test.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ tests:
             template:
               metadata:
                 annotations:
-                  checksum/config: 8ce33705643160d06b4062deacb00ee60e290dc4bd9069053b81dcac91279785
+                  checksum/config: 89183888d634af5f963ae86d045f36b06026324e3540e15c8cc8e292045a7ce3
```
Comment (v1v) [inline charts/jenkins/tests/jenkins-controller-statefulset-test.yaml:47]: I've no clue why I need to do this...

**Comment by v1v** [comment]
cc @timja 🙏 and thanks :)

File: charts/jenkins/tests/jenkins-controller-statefulset-test.yaml (hunk 1 above)
Comment (timja) [reply charts/jenkins/tests/jenkins-controller-statefulset-test.yaml:47]: it will be config that needs to restart the controller on change

**Review by timja — APPROVED** [review]
it will do till the plugin manager is enhanced to not throw when compatible and not set on the top level

**Pull request merged**